libs/ReinforcerKit/tests/ReinforcerKit_test.cpp

```diff
@@ -55,26 +48,20 @@ class ReinforcerkitTest : public ::testing::Test
 
 	ReinforcerKit reinforcerkit;
 
-	void expectedCallsMovingReinforcer()
+	void expectedCallsMovingReinforcer(interface::LEDAnimation *animation)
 	{
 		EXPECT_CALL(mock_videokit, playVideoOnce);
 		EXPECT_CALL(mock_motor_left, stop).Times(1);
 		EXPECT_CALL(mock_motor_right, stop).Times(1);
 		EXPECT_CALL(mock_motor_left, spin).Times(1);
 		EXPECT_CALL(mock_motor_right, spin).Times(1);
```

Comment:
not directly related but:

this should be tested with mock::MotionKit now that Reinforcer use MotionKit

this will make everything cleaner -- no need to test the motors are spinning individually when you have the MotionKit::rotate function